files/ja/web/javascript/reference/statements/for...in/index.html

```diff
@@ -142,7 +142,7 @@ <h2 id="See_also" name="See_also">関連情報</h2>
  <li>{{jsxref("Statements/for...of", "for...of")}} - プロパティ値に対して繰り返す同様の文</li>
  <li>{{jsxref("Statements/for_each...in", "for each...in")}} - <code>for...in</code> に似ていますが、プロパティ名そのものではなく、オブジェクトのプロパティの値に対して反復します。(廃止されました)</li>
  <li>{{jsxref("Statements/for", "for")}}</li>
- <li><a href="/ja/docs/Web/JavaScript/Guide/Iterators_and_Generators">ジェネレーター式</a> (<code>or...in</code> を使っている)</li>
+ <li><a href="/ja/docs/Web/JavaScript/Guide/Iterators_and_Generators">ジェネレーター式</a> (<code>for...in</code> を使っている)</li>
```

Comment:
一旦 `f` がなくなっているという typo だけ修正しましたが、en-US を見たところ次のように書かれていました

https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Statements/for...in

```
Iterators and Generator functions (usable with for...of syntax)
```

とすると、次のように内容も直すほうが良いかもしれません。

```suggestion
 <li><a href="/ja/docs/Web/JavaScript/Guide/Iterators_and_Generators">ジェネレーター式</a> (<code>for...of</code> とあわせて利用できます)</li>
```